### ISSUE TITLE
docs.solanalabs.com defaults to a non-stable release causing multiple problems for Solana developers

### ISSUE THREAD

**Issue by mikemaccana** [anza-xyz/agave#679]
#### Problem

At Solana Foundation Dev Ecosystem, we have people in Stack Overflow and our live training complaining about multiple issues (eg `unknown feature proc_macro_span_shrink`) that are caused by non-stable Solana CLI releases. This is because https://docs.solanalabs.com/cli/install does not point to the stable release by default: 

> Install the Solana release [v1.18.4](https://github.com/solana-labs/solana/releases/tag/v1.18.4) on your machine by running:
> sh -c "$(curl -sSfL https://release.solana.com/v1.18.4/install)"

It should read:

> Install the Solana release [v1.17.25](https://github.com/solana-labs/solana/releases/tag/stable) on your machine by running:
> sh -c "$(curl -sSfL https://release.solana.com/stable/install)"

Or similar.

Separately, and maybe related: 1.18.8 is out right now but the website still points to 1.18.4. Not sure if that's the desired behaviour either.

#### Proposed Solution

Originally I proposed https://github.com/anza-xyz/agave/pull/577 but it wasn't totally clear there were multiple instances of the docs site and I'm not familiar with the docs CI process enough to fix it. However someone with a little more knowledge of the Anza CI should hopefully be able to fix this easily.

**Comment by t-nelson** [comment]
@yihau can you take a look? i'm guessing we missed something in the release mirror logic to trigger docs update

**Comment by CriesofCarrots** [comment]
@t-nelson , I think we've used the latest beta version in docs.solana.com for years.

**Comment by yihau** [comment]
yeah, I found that the docs pipeline has been disabled in Solana repo. I just turned it on. it should publish a new page in our next release 🫠

**Comment by t-nelson** [comment]
> @t-nelson , I think we've used the latest beta version in docs.solana.com for years.

yep, mainly concerned that 1.18.11 is out and we're still suggesting 1.18.4

> yeah, I found that the docs pipeline has been disabled in Solana repo. I just turned it on. it should publish a new page in our next release 🫠

that'll do it

---

@mikemaccana can you elaborate on where that error comes from?

**Comment by mikemaccana** [comment]
Thanks @t-nelson ! Re: "can you elaborate on where that error comes from?" - do you mean the `proc_macro_span_shrink` error? My recollection is it was just occurring with Anchor 0.29 on Solana 1.18, during `anchor build`, on a new blank project made with `anchor init`. I'm in the middle of trying to get Anchor 0.30 working right now but can check it out further if needed later.

**Comment by t-nelson** [comment]
ah got it. we're not going to change the recommended cli version for an ephemeral anchor incompatibility

once the cli bins are installed, you can switch to another version trivially with `solana-install init vX.Y.Z`. probably best to add a step to the training content to do that until the incompatibility is resolved. teach them to fish or w/e

**Comment by steviez** [comment]
This issue is > 2 years old; closing out. Can re-open with current versions if still applicable
